paper/paper.md

```diff
@@ -313,29 +321,32 @@ the future. For example, the cyclic identification algorithm (`ioID`)
 realistic graphs that contain cycles, such as how biomolecular signaling
 pathways often contain feedback loops. Further, missing data identification
 algorithms can account for data that is missing not at random (MNAR) by modeling
-the underlying missingness mechanism [@mohan2021]. Several algorithms noted in
-the review by @JSSv099i05, such as generalized identification (`gID`)
-[@lee2019general] and generalized counterfactual identification (`gID*`)
-[@correa2021counterfactual], can be formulated as special cases of
-counterfactual transportability. Therefore, we plan to improve the user
-experience by exposing more powerful algorithms like counterfactual transport
-through a simplified APIs corresponding to special cases like `gID` and `gID*`.
-Similarly, we plan to implement probabilistic expression simplification
-[@tikka2017b] to improve the consistency of the estimands output from
-identification algorithms.
-
-It remains an open research question how to estimate the causal effect for an
-arbitrary estimand produced by an algorithm more sophisticated than `ID`. Two
-potential avenues for overcoming this might be a combination of the Pyro
-probabilistic programming langauge [@bingham2018pyro] and its causal inference
-extension [ChiRho](https://github.com/BasisResearch/chirho). Tractable circuits
-[@darwiche2022causalinferenceusingtractable] also present a new paradigm for
-generic estimation. Such a generalization would be a lofty achievement and
-enable the automation of downstream applications in experimental design.
+the underlying missingness mechanism [@mohan2021]. Implementing recent
+algorithms that provide sufficient conditions for identification in hierarchical
+causal models [@weinstein2024hierarchicalcausalmodels] would be useful for
+supporting causal identification in probabilistic programming languages, such as
+ChiRho. Several algorithms noted in the review by @JSSv099i05, such as
+generalized identification (`gID`) [@lee2019general] and generalized
+counterfactual identification (`gID*`) [@correa2021counterfactual], can be
+formulated as special cases of counterfactual transportability. Therefore, we
+plan to improve the user experience by exposing more powerful algorithms like
+counterfactual transport through a simplified APIs corresponding to special
+cases like `gID` and `gID*`. Similarly, we plan to implement probabilistic
+expression simplification [@tikka2017b] to improve the consistency of the
+estimands output from identification algorithms.
+
+@agrawal2024automated recently demonstrated automatically generating an
```

Comment:
I think keeping the first framing sentence gives important context on why we're talking about this 

> It remains an open research question how to estimate the causal effect for an
arbitrary estimand produced by an algorithm more sophisticated than `ID`.